pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -50,10 +57,13 @@ type capability struct {
 }
 
 var _ capabilityIface = (*capability)(nil)
+var _ capabilities.ConsensusCapability = (*capability)(nil)
```

Comment:
@HenryNguyen5 How come you added this? I think this should do the same job as capabilityIface (my intent was to loosely couple this to the consensus interface so we only pulled in what we needed)

Reply:
No, not necessarily -- but in that case I think you don't need line 59, so I'd prefer we pick one and remove the other

Reply:
I'm confused since capabilityIface contains completely separate methods (used to implement a reporting plugin) whereas ConsensusCapability is to make sure that this struct really is a capability.

I removed the line I added for now. I'd love to understand why we don't enforce the intersection of the two interfaces against this struct.